envdir/env.py

```diff
@@ -36,11 +39,13 @@ def __enter__(self):
     def __exit__(self, type, value, traceback):
         self.clear()
 
-    def __getitem__(self, name):
+    def __getitem__(self, name, default=_sentinel):
         try:
-            return self._get(name)
-        except _EmptyFile:
-            pass
+            return self._get(name, default=default)
+        except (_EmptyFile, FileNotFoundError):
```

Comment:
But `IOError` is available in Python 3, too - and might be better to catch other errors related to it?